drivers/watchdog/wdt_wwdg_stm32.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019 Centaur Analytics, Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_WATCHDOG_WWDG_STM32_H_
+#define ZEPHYR_DRIVERS_WATCHDOG_WWDG_STM32_H_
+
+#include <zephyr/types.h>
+
+/**
+ * @brief Driver for System Window Watchdog (WWDG) for STM32 MCUs
+ *
+ * The driver targets all STM32 SoCs. For details please refer to
+ * an appropriate reference manual and look for chapter called:
+ *
+ *   System window watchdog (WWDG)
+ *
+ */
+
+/* driver data */
+struct wwdg_stm32_data {
+	/* WWDG peripheral instance. */
+	WWDG_TypeDef *Instance;
```

Comment:
Please set this field in a ROMable _config structure, along with clock settings.